### DISCUSSION TITLE
Feature request: Ability to save drafts that can persist beyond local storage

### DISCUSSION THREAD

**Discussion by cointastical** [stackernews/stacker.news#3006] [Ideas]
My request is for a method to explicitly be able to click Save for my draft so that I can recover it across sessions, (or maybe a periodic automatic save to SN's back-end, like after every ten seconds following edit activity, or something like that).  SN does now have [protection against accidental navigation (e.g., closing the tab)](https://github.com/stackernews/stacker.news/issues/69), but that doesn't help if I close my browser (and I am using Incognito mode/private browser).

Discussed further here: https://stacker.news/items/99262

**Comment by cointastical** [comment]
Also possibly related is Issue #121.

**Comment by ekzyis** [comment]
Bumping this since I think this would be a great feature for content creators / people who post long-form content on SN.

Even though I am not a dedicated content creator on SN, I already have sometimes the need to save posts on my local disk such that I can start writing another post without the other one getting overwritten. 

Basically, I want to be able to polish a post before submitting it without blocking other posts I might come up with in the meantime.

Having the option to save drafts would fix this need.

This also means that you can have multiple drafts and overwrite them with a new version.
